egs/wsj/s5/steps/nnet3/xconfig_to_configs.py

```diff
@@ -159,6 +159,9 @@ def write_config_files(config_dir, all_layers):
     for layer in all_layers:
         try:
             pairs = layer.get_full_config()
+            if layer.descriptors.has_key('input') and 'input' in layer.descriptors['input']['final-string']:
```

Comment:
Before, there must be FixedAffineLayer in xconfig, otherwise the 'output-node' would not appear in init.config, as 'output-node' is generated by get_full_config() of 'class FixedAffineLayer'.

Now, the generation of 'output-node' was moved into the final stage of xconfig, no matter whether there is FixedAffineLayer or not.